app/assets/javascripts/discourse/app/components/user-preferences/user-passkeys.js

```diff
@@ -0,0 +1,143 @@
+import I18n from "I18n";
+import { bufferToBase64, stringToBuffer } from "discourse/lib/webauthn";
+import RenamePasskey from "discourse/components/user-preferences/rename-passkey";
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+
+export default class UserPasskeys extends Component {
+  @service dialog;
+  @service currentUser;
+  @service capabilities;
+
+  get isCurrentUser() {
+    return this.currentUser.id === this.args.model.id;
+  }
+
+  get passkeyName() {
```

Comment:
Same here, I don't think this needs to be a getter and can just be a normal property that is computed once.